src/libraries/System.IO.FileSystem/tests/File/Open.cs

```diff
@@ -72,8 +70,7 @@ protected override FileStream CreateFileStream(string path, FileMode mode, FileA
                     Access = access,
                     Share = share,
                     Options = options,
-                    BufferSize = bufferSize,
-                    PreallocationSize = PreallocationSize
+                    BufferSize = bufferSize
```

Comment:
Why is PreallocationSize being removed from all of these tests?

Reply:
The tests would no longer pass for modes other than `FileMode.Create*` or `FileMode.Truncate`.

Reply:
> The tests would no longer pass for modes other than FileMode.Create* or FileMode.Truncate.

But were we relying on these for coverage in the cases that were supported?  If not, fine.  If yes, I'm wondering if we now need to make the ctor used by these tests conditional based on whether a non-negative preallocation size is provided.

Reply:
I'll take a closer look at the coverage.